runtime/mgmt/mgmtinit.c

```diff
@@ -31,3 +33,48 @@ JNI_OnLoad(JavaVM *vm, void *reserved)
 {
 	return JNI_VERSION_1_8;
 }
+
+/**
+ * Determines whether file path has user permissions only.
+ *
+ * @param path file path
+ * @return Returns false on Windows and true if the file has only user permissions,
```

Comment:
I don't see the code to make it return false on Windows....

Reply:
In j9file_stat -> omrfile_stat in win32 the permission bit setting is granular enough for this method: https://github.com/eclipse/openj9-omr/blob/master/port/win32/omrfile.c#L1020

Reply:
*is not